CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contribution Guide
+
+jargonLint aims to be accessible to all developers and writers, regardless of prior experience. However, we ask that you follow this guide in your contributions. This helps us support consistency and structure within the repository.
+
+## Simple Fixes
+
+There can be simple typos or edits to only a few lines that you want to correct.
+
+In this case, you can simply press the "edit" option in the upper page and GitHub should guide you through:
+1. Creating a fork
+2. Editing the fork
+3. Opening a pull request
+
+If the GitHub suggestions you should see on the upper area are not satisfactory, and you have limited experience, you can simply open an issue for your fix. 
+
+## Adding New Rules
+
+We aim to add rules that are comprehensive and generally useful. Therefore, we cannot accept opinionated rules or those with a limited use case. 
```

Comment:
I didn't want to add too much since it's also mentioned in the README to the project.